x-pack/solutions/security/plugins/security_solution_serverless/public/upselling/register_upsellings.test.tsx

```diff
@@ -79,4 +80,14 @@ describe('registerUpsellings', () => {
     expect(setMessages).toHaveBeenCalledTimes(1);
     expect(setMessages).toHaveBeenCalledWith(expectedMessagesObject);
   });
+
+  describe('upsellingPages', () => {
+    it('should register the `Translated rules` page with expect pli', () => {
+      const translatedRulesPage = upsellingPages.find(
+        ({ pageName }) => pageName === SecurityPageName.siemMigrationsRules
+      );
+
+      expect(translatedRulesPage?.pli).toEqual(ProductFeatureKey.siemMigrations);
+    });
+  });
```

Comment:
NIT: There's no need to test the upsellingPages configuration has specific settings. There's no logic to test, it's just declarative code.